source/Calamari.Tests/ArgoCD/Git/GitVendorApiAdapters/GitHubApiAdapterFactoryTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Calamari.ArgoCD.Git;
+using Calamari.ArgoCD.Git.GitVendorApiAdapters;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Calamari.Tests.ArgoCD.Git.GitVendorApiAdapters
+{
+    [TestFixture]
+    public class GitHubApiAdapterFactoryTests
```

Comment:
Can probably add more tests for the other factories, but this one is just reusing some exists that were around for the prior GitHub specific logic